README.md

```diff
@@ -29,17 +29,32 @@ marks](CaliptraTrademarkPolicy.md).
 * [FMC](https://github.com/chipsalliance/caliptra-sw/tree/main/fmc#readme)
 * [Firmware](https://github.com/chipsalliance/caliptra-sw/blob/main/runtime/README.md)
 
+## Security policy
+
+For information on the Caliptra security policy, how to report a vulerability,
+and published security advisories, refer to
+[Caliptra security policy](https://github.com/chipsalliance/caliptra/security/policy).
+
+### Product creator registration
+
+Companies or individuals building products that integrate Caliptra are
+encouraged to register as a Caliptra product creator. Product creators will
+automatically get early notification of embargoed security vulnerabilities, as
+defined in the Caliptra security policy.
+
+* **Registration process to be defined**
+
 ## Repositories
 
 The Caliptra code base and documentation are split across five repositories:
 
-| Repo | What |      |
+| Repo | What | Description |
```

Comment:
`Repo`-->`Repository` throughout this section
`What`-->`Purpose`

Reply:
I think neither word (What or Purpose) describes this properly. How about "Areas of interest"?